### PR TITLE
run models with tmmn and tmmx

### PR DESCRIPTION
**Pull request by galengorski**
This PR is in reference to #153 , running all four models with tmmx and tmmn. The PR updates the config files and the weights from the model runs

### CODE REVIEW

File: 2a_model/src/models/1_metab_multitask/2b_multitask_do_gpp.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ exp_name: "1b_multitask_do_gpp"
 
 site_set: "well_obs"
```
Comment (lekoenig) [inline 2a_model/src/models/1_metab_multitask/2b_multitask_do_gpp.yml:3]: Not related to "tmmx" or "tmmn", but is there a reason we use "well_obs" for `site_set` here, but "med_obs" for the others?

**Review by lekoenig — APPROVED** [review]
Looks good, @galengorski! I added just one (unrelated) question about the model config files. And I'm curious - can you tell if adding `tmmn` had any impact on the model performance?

**Comment by lekoenig** [comment]
> And I'm curious - can you tell if adding tmmn had any impact on the model performance?

I now see that you've included a few plots in #152 😊

**Review by jsadler2 — APPROVED** [review]
Looks good to me.

**Pull request merged**